dot/node_integration_test.go

```diff
@@ -120,7 +120,11 @@ func TestNewNode(t *testing.T) {
 		return stateSrvc, nil
 	})
 
-	phs, err := parachain.NewService(testNetworkService, "random_fork_id", common.Hash{})
+	mockBlockState := &state.BlockState{}
+	mockStateService := &state.Service{
+		Block: mockBlockState,
+	}
+	phs, err := parachain.NewService(testNetworkService, "random_fork_id", mockStateService)
```

Comment:
Just curious, what does `phs` stand for?

Reply:
Good question, I'm not sure, `phs` was defined before I modified this code.

Reply:
parachain host service?